cmd/blocktx.go

```diff
@@ -144,3 +155,29 @@ func StartBlockTx(logger *slog.Logger) (func(), error) {
 		peerHandler.Shutdown()
 	}, nil
 }
+
+func StartHealthServerBlocktx(serv *blocktx.Server) error {
+	gs := grpc.NewServer()
+	defer gs.Stop()
+
+	grpc_health_v1.RegisterHealthServer(gs, serv) // registration
+	// register your own services
+	reflection.Register(gs)
+
+	address, err := config.GetString("blocktx.blocktxHealthServerDialAddr") //"localhost:8005"
```

Comment:
According to the other comment https://github.com/bitcoin-sv/arc/pull/297/files#r1494276927
this could then be changed to `blocktx.healthServerDialAddr`